src/App.js

```diff
@@ -39,18 +41,27 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
-  }
+  onItemClicked = (type) => {
+    console.log('in the app type:')
+    console.log(type);
+
+    //wave 3
+    let totalPoints = this.state.points + 1;
+
+    if (type === "litter") {
+      this.setState({ points: totalPoints });
```

Comment:
One issue, if I click on the same piece of litter multiple times, I can run up the score!